Volodymyr_Kuksa/0/custom_range.py

```diff
@@ -0,0 +1,42 @@
+def custom_range(*args):
+    """
+    Yield sequence of integers, generated according to arguments.
+
+    Possible prototypes:
+    custom_range(stop)
+    custom_range(start, stop)
+    custom_range(start, stop, step)
+
+    :param args:    start - starting value of the resulting sequence (inclusive, defaults to 0).
+                    stop - ending value of the resulting sequence (exclusive).
+                    step - step of the sequence (defaults to 0).
+    :type args: int.
+
+    :yield: sequence of integers.
+    :yield type: int.
```

Comment:
I haven't found any info on yields in docstrings, are there any 'Best Practices' for documenting that?